src/CMakeLists.txt

```diff
@@ -9,4 +9,5 @@ add_executable ("${PROJECT_NAME}" main.cpp)
 target_link_libraries ("${PROJECT_NAME}" PRIVATE "${PROJECT_NAME}_lib")
 
 # Include code-coverage settings:
-target_link_libraries ("${PROJECT_NAME}" PUBLIC seqan_app_coverage_config)
+target_link_libraries ("${PROJECT_NAME}_lib" PUBLIC seqan_app_coverage)
```

Comment:
I would like that to be separate from a normal source build.

Can't we move this ....